ReleaseProcessScript/tools/PesterTester/git_base_functions.Tests.ps1

```diff
@@ -175,4 +175,18 @@ Describe "git_base_functions" {
       { Check-Branch-Up-To-Date "master" } | Should Not Throw 
     }
   }
+
+  Context "Get-Ancestor" {
+    It "should print all found ancestors if multiple are found" {
+      git checkout -b "additionalAncestor" --quiet
+      git checkout -b "newBranch" --quiet
+      $ExpectedAncestors = "master", "additionalAncestor"
+      Mock Write-Host -Verifiable -ParameterFilter {$Object -eq "We expected to find one of following Ancestors: 'master additionalAncestor', but found multiple possible Ancestors."}
```

Comment:
Hmm...drop single quote. we're printing multiple individual entries which are only whitespace separated inside a block delimited by a colon and a comma. The extra quotes don't help if there is an actual problem and might imply that the printed text is a single item instead of multiple. The completely correct way would be to place each item into its own quotes but that might be too expensive, codewise in powershell.

Reply:
I managed to do a "correcter" way since I found a StackOverflow answer for it